src/app/tasks/task.model.ts

```diff
@@ -5,6 +5,7 @@ export type Option = {
 }
 
 export interface Task {
+    id: number | null,
```

Comment:
I don't think ID will ever be null, when you implement BE each task item will have a generated ID